pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -413,7 +413,7 @@ func getDefaultKieServerImage(product string, config *v1.CommonConfig, from *cor
 	if from != nil {
 		return *from
 	}
-	imageName := fmt.Sprintf("%s%s-kieserver-openshift:%s", product, config.Version, constants.ImageStreamTag)
+	imageName := fmt.Sprintf("%s%s-kieserver-openshift:%s", product, config.Version, config.ImageTag)
```

Comment:
@bmozaffa can we limit this PR against 1.0.x branch to only this change? i'm thinking we should avoid an sdk upgrade in that branch as its only for the 7.3.x releases.

Reply:
Other changes removed, ready for review